Docs: add text formatting rules (#772)
grafana/writers-toolkit: docs/sources/write/style-guide/style-conventions/index.md

### Bold

Use bold formatting (`**`) when directly referring to UI elements.

> When referring to an abstract UI element such as a role (Admin, Editor, Viewer) without directly referencing the UI, capitalize, but don't bold the word.

Author: ```suggestion
When you're referring to an abstract UI element, such as a role (Admin, Editor, Viewer), without directly referencing the UI, don't bold the word; just capitalize it.
```

> It's okay to use bold inline with other prose in a table, but don't use bold for the all text in the leading column, even if those items are UI elements.

Author: ```suggestion
It's okay to use bold inline with other prose in a table, but don't use bold for the all text in the first column, even if it's a table of UI elements.
```

> It's okay to use bold for a the first sentence in unordered lists that are followed by more information.

Reviewer: Battle of the OK commences :D https://en.wikipedia.org/wiki/OK

I'm OK with either, with a preference for OK even though I guess it's a bit weird looking.

Google style doesn't have a documented opinion in their word list that I can find but seems to use okay in their sentences.

Reviewer: So perhaps we should say okay is the way and I'll stop adding more OKs to our docs
